runlarky/src/main/proto/larky.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package com.verygood.security.mode.grpc;
+option java_outer_classname = "LarkyProto";
+option java_multiple_files = true;
+
+message LarkyProcessRequest {
+  string script = 1;
+  bytes input = 2;
```

Comment:
the interface will also include string & Starlarky request / response structs soon